.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
**issue:** Please don't make a library strict unless it is motivated by a minimal support for the required features.

Reply:
agree, removed